src/workloads/execution/SlidingWindows.yml

```diff
@@ -0,0 +1,138 @@
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {avg: {$avg: {input: "$temp", range: [-10, 0], unit: "second"}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: MinPositionBased
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {min: {$min: {input: "$temp", documents: [-5, 5]}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: MinTimeBased
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {min: {$min: {input: "$temp", range: [-20, 0], unit: "second"}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: ExpMovingAvgSmallN
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {ema: {$expMovingAvg: {input: "$temp", N: 10}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: ExpMovingAvgLargeN
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {ema: {$expMovingAvg: {input: "$temp", N: 500}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: DerivativeSmallWindow
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {rate: {$derivative: {input: "$temp"}, window: {documents: [-10, 0]}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+    - OperationMetricsName: DerivativeLargeWindow
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {rate: {$derivative: {input: "$temp"}, window: {documents: [-500, 0]}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
+
+AutoRun:
+  Requires:
+    mongodb_setup:
+    - standalone
+    - replica
+    - replica-all-feature-flags
```

Comment:
Can we add shard-lite here? Also I don't think we need the -all-feature-flags version given WF is on by default, but I guess it can't hurt?